dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -18,3 +18,12 @@ cdi12.desc=Defines the behavior of the Contexts and Dependency Injection (CDI) v
 
 cdi12.enableImplicitBeanArchives.name=Enable scanning implicit bean archives
 cdi12.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for any bean discoveries.
+
+cdi=Contexts And Dependency Injection
+cdi.desc=Defines the behavior of the Contexts and Dependency Injection.
+
+cdi.enableImplicitBeanArchives.name=Enable scanning implicit bean archives. Applies to CDI 1.2 or newer.
```

Comment:
unless there is a precedent to use "newer", I recommend using "later" for consistency with other docs/messages.

suggested edit:

Enable scanning of  implicit bean archives. This property applies to CDI 1.2 or later.

Reply:
or if not a "property", substitute whatever entity applies

Reply:
The existing element description for `cdi12` says "v1.2 or newer".
https://github.com/OpenLiberty/open-liberty/blob/a17777d29dd937ab795a3c452f70189be67517a6/dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties#L17

Reply:
So changing this to
`Enable scanning implicit bean archives. This property applies to CDI v1.2 or newer.`